OPC user edit throwing hydrate errors fixes (#54)
chef/knife-opc: README.md

Deletes the given OPC user.

### knife opc user edit USERNAME (options)

> - `-i FILENAME`, `--input FILENAME`: Will read the user information (i.e. username, email, display_name, first_name, last_name, middle_name) from the input FILENAME and update the given OPC user.

Reviewer: ```suggestion
- `-i FILENAME`, `--input FILENAME`: Reads the user information (`username`, `email`, `display_name`, `first_name`, `last_name`, `middle_name`) from the input FILENAME and updates the given OPC user.
```

Author: Agreed,I  updated it.

> - `-f FILENAME`, `--filename FILENAME`: Open FILENAME on $EDITOR. When finished, Updates the given OPC user.

Reviewer: ```suggestion
- `-f FILENAME`, `--filename FILENAME`: Opens FILENAME in $EDITOR. Updates the given OPC user when the file is saved.
```

Reviewer: Is the update on save or close?

Author: @kagarmoe Update is on save and close the $Editor.

Deletes the given OPC user.

### knife opc user edit USERNAME (options)

> - `-i FILENAME`, `--input FILENAME`: Reads the user information (`username`, `email`, `display_name`, `first_name`, `last_name`, `middle_name`) from the input FILENAME and updates the given OPC user.

Reviewer 2: It would be helpful to mention that this file is expected to be in the json format and optionally add an example.

Author: Agreed, I updated it.

Deletes the given OPC user.

### knife opc user edit USERNAME (options)

> - `-i FILENAME`, `--input FILENAME`: Reads the user information (`username`, `email`, `display_name`, `first_name`, `last_name`, `middle_name`) from the input FILENAME and updates the given OPC user. This file is expected to be in the json format.

Reviewer 3: ```suggestion
- `-i FILENAME`, `--input FILENAME`: Reads the user information (`username`, `email`, `display_name`, `first_name`, `last_name`, `middle_name`) from the input FILENAME and updates the given OPC user. Knife will accept a JSON file.
```

Author: @IanMadd I updated the changes.Please have a look.
